### ISSUE TITLE
Roboto sans font does not seem to be used anymore

### ISSUE THREAD

**Issue by armenzg** [mozilla/bugzilla-dashboard#36]
It used to work but it does not seem to work anymore:
https://github.com/mozilla/bugzilla-dashboard/blob/master/package.json#L38

**Comment by armenzg** [comment]
@TrangEmily this seems to have stopped working if interested.

**Comment by TrangEmily** [comment]
Hi @armenzg How do you know it stopped working? I want to work with it but don't see any error on console logs. Thanks.

**Comment by armenzg** [comment]
Good question!

I inspected the page and the text showing is "Roboto", however, I was expecting the sans serif version of it.
I might have never noticed that I added Roboto instead of the san serif version of it.

I think the problem lays in not having this as the font-family `font-family:'Roboto',Open Sans,sans-serif;`
https://github.com/mozilla/bugzilla-dashboard/blob/626af8c429e6188eda3801e624f5a8161a51a3ab/src/App/index.jsx#L11
